test/fixtures/rules/std_rule_cases/L025.yml

```diff
@@ -43,6 +43,18 @@ test_ignore_postgres_value_table_functions:
     core:
       dialect: postgres
 
+test_ignore_postgres_value_table_functions_generate_series:
```

Comment:
Would it be worth adding an opposite test, like a similar query but a different function name?

Reply:
Don't think we should put a random, gibberish function in can't think of a function that you could put in a `FROM` clause without it being a value table function.

Maybe all functions in FROM clauses should be value table functions rather than explicitly listing them? But let's keep this change simple since we're just about to release so that's out of scope for now.